src/pages/developer-tools/docker/index.md

```diff
@@ -12,10 +12,57 @@ Similar to Github, DockerHub offers a wide variety of images (Ubuntu and others)
 ### See also
 - Docker Compose : to create and manage several containers at the same time.
 
-### Links
+
+#### More Information:
 You can find plenty of informations in the following sites : 
 - <a href='https://www.docker.com/' target='_blank' rel='nofollow'>Docker website</a>
 - <a href='https://docs.docker.com/' target='_blank' rel='nofollow'>Docker docs</a>
 - <a href='https://hub.docker.com/' target='_blank' rel='nofollow'>DockerHub</a>
+[logo]: https://www.docker.com/sites/default/files/Whale%20Logo332_5.png "Docker"
+
+
+## Docker
+
+Docker is an open source project to automate the dployment of any software applications inside containers.  A container image is a lightwieght, stand-alone, execuateble package of a piece of software that includes everything need to run it.
+
+Docker is available for both Windows and Linux environment. Docker container running on a single machine share that machine's oprating system kernel. Docker container isolates applications from one another. 
+
+
+### Container Vs. Virtual Machines
+
+* Container virtualises the Operating System and more portable, where as VMs virtualises the Hardware. 
+
+* Container are an abstraction at the app layer that packages code and dependencies togehter. VMs are an abstraction of physical hardware turning one server into many. Hypervisor helps VM to do so.
+
+* [CONTAINERS ARE NOT VMS](https://blog.docker.com/2016/03/containers-are-not-vms/)
+
+
+### Installation of Docker
+
+Docker is available in two editions: Community Edition (CE) and Enterprise Edition (EE).
+
+[Install it from here](https://docs.docker.com/engine/installation/)
+
+Once installed try this
+
+$ docker run hello-world
+
+Hello from Docker!
+This message shows that your installation appears to be working correctly.
+
+
+### Where to use Docker
+
+* [8 Proven Real-World Ways to Use Docker](https://www.airpair.com/docker/posts/8-proven-real-world-ways-to-use-docker)
+
+* [What is Docker and When to Use It](https://www.ctl.io/developers/blog/post/what-is-docker-and-when-to-use-it/)
+
+### Docker Tutorials
```

Comment:
move these links under 

`#### More Information:` 

and make these as anchor links